### DISCUSSION TITLE
Craftable amethyst shards

### DISCUSSION THREAD

**Discussion by JSkellington0119** [BedrockTweaks/Files#365] [Ideas]
If possible but I would like to see amethyst blocks convert to shards because seeing that shards to blocks but not vise versa makes no sense

**Comment by AtlasOfEden** [comment]
Same with Nether Warts and Nether Quartz

**Reply by JSkellington0119** [reply]
I thought there was one for nether warts but not nether quartz though

**Comment by SquatchHunter** [comment]
iirc I have packs like these I made for personal use
however community packs will be done after addons are
we still dont have a timeline
